core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -934,15 +957,16 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
      * @memberof DotEmaDialogComponent
      */
     private handleCopyContent(): Observable<DotCMSContentlet> {
+        // console.log('we should reload store');
+
         return this.dotCopyContentService.copyInPage(this.currentTreeNode()).pipe(
             catchError((error) =>
                 this.dotHttpErrorManagerService.handle(error).pipe(
                     tap(() => this.dialog.resetDialog()), // If there is an error, we set the status to idle
                     map(() => null)
                 )
             ),
-            filter((contentlet: DotCMSContentlet) => !!contentlet?.inode),
-            tap(() => this.reloadIframe()) // If the contentlet is copied, we reload the iframe
+            filter((contentlet: DotCMSContentlet) => !!contentlet?.inode)
```

Comment:
What happened here?

Reply:
The manual call to reloadIframe() is unnecesary, this method is called in handleEditContentlet()
And both ways on this method results on savePage to update the loading state and trigger the reload